source/ranges.tex

```diff
@@ -3202,6 +3210,430 @@
+           @\libconcept{constructible_from}@<Bound, BoundArgs...>
+constexpr explicit repeat_view(piecewise_construct_t,
+  tuple<Wargs...> value_args, tuple<BoundArgs...> bound_args = tuple<>{});
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes \exposid{value_} with arguments of types \exposid{WArgs...}
+obtained by forwarding the elements of \tcode{value_args} and
+initializes \exposid{bound_} with arguments of types \tcode{BoundArgs...}
+obtained by forwarding the elements of \tcode{bound_args}.
+(Here, forwarding an element \tcode{x} of type \tcode{U} within a tuple object
+means calling \tcode{std::forward<U>(x)}.)
+\end{itemdescr}
+
+\indexlibrarymember{begin}{repeat_view}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ begin() const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{iterator}(addressof(*value_));}
+\end{itemdescr}
+
+\indexlibrarymember{end}{repeat_view}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ end() const requires (!@\libconcept{same_as}@<Bound, unreachable_sentinel_t>);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{iterator}(addressof(*\exposid{value_}), \exposid{bound_});}
+\end{itemdescr}
+
+\indexlibrarymember{end}{repeat_view}%
+\begin{itemdecl}
+constexpr unreachable_sentinel_t end() const noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return unreachable_sentinel;}
+\end{itemdescr}
+
+\indexlibrarymember{size}{repeat_view}%
+\begin{itemdecl}
+constexpr auto size() const requires (!same_as<Bound, unreachable_sentinel_t>);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{to-unsigned-like}(\exposid{bound_});}
+\end{itemdescr}
+
+\rSec3[range.repeat.iterator]{Class template \tcode{repeat_view::\exposid{iterator}}}
+
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{move_constructible}@ W, @\libconcept{semiregular}@ Bound = unreachable_sentinel_t>
+    requires (is_object_v<W> && @\libconcept{same_as}@<W, remove_cv_t<W>> &&
+              (@\exposid{is-integer-like}@<Bound> || @\libconcept{same_as}@<Bound, unreachable_sentinel_t>))
+  class repeat_view<W, Bound>::@\exposid{iterator}@ {
+  private:
+    using @\exposid{index-type}@ =                  // \expos
+      conditional_t<@\libconcept{same_as}@<Bound, unreachable_sentinel_t>, ptrdiff_t, Bound>;
+    const W* @\exposid{value_}@ = nullptr;          // \expos
+    @\exposid{index-type}@ @\exposid{current_}@ = @\exposid{index-type}@(); // \expos
+
+    constexpr explicit @\exposid{iterator}@(const W* value, @\exposid{index-type}@ b = @\exposid{index-type}@());   // \expos
+
+  public:
+    using iterator_concept = random_access_iterator_tag;
+    using iterator_category = random_access_iterator_tag;
+    using value_type = W;
+    using difference_type = conditional_t<@\exposid{is-signed-like}@<@\exposid{index-type}@>,
```

Comment:
What is _`is-signed-like`_? Should this be _`is-signed-integer-like`_? I suspect this is a typo in the paper.

Reply:
Yes, I think so, thanks!